doc/sphinx/changes.rst

```diff
@@ -8,6 +8,978 @@ Recent changes
+  * Fix the Dune files as to copy the include files into the `_build` directory
+  * Add a custom OCaml toplevel directory `#go` which does the same action as `go ()`, but with a more native syntax
+  * Make the include files not modify the global flags permenantly
+  * Rename `Flags.in_toplevel` to `Flags.in_ml_toplevel` to highlight the fact that it is about the OCaml toplevel and not the Coq one
+  * Rename `dev/shim/coqbyte` to `dev/shim/coqtop.byte` to match the standard installation convention and make it easier to find
+  * Update the debugging documentation, fix formatting there
+  * Add a new `coq_config_bin.ml` file, and also unify how coqtop and coqc are built
+  (`#18771 <https://github.com/coq/coq/pull/18771>`_,
+  by Anton Danilkin).
+- **Removed:**
+  Support for ``.vio`` files and for ``.vio2vo`` transformation has
+  been removed, compilation to ``.vos`` is the supported method for
+  quick compilation now
+  (`#18424 <https://github.com/coq/coq/pull/18424>`_,
+  fixes `#4007 <https://github.com/coq/coq/issues/4007>`_
+  and `#4013 <https://github.com/coq/coq/issues/4013>`_
+  and `#4123 <https://github.com/coq/coq/issues/4123>`_
+  and `#5308 <https://github.com/coq/coq/issues/5308>`_
+  and `#5223 <https://github.com/coq/coq/issues/5223>`_
+  and `#6720 <https://github.com/coq/coq/issues/6720>`_
+  and `#8402 <https://github.com/coq/coq/issues/8402>`_
+  and `#9637 <https://github.com/coq/coq/issues/9637>`_
+  and `#11471 <https://github.com/coq/coq/issues/11471>`_
+  and `#18380 <https://github.com/coq/coq/issues/18380>`_,
+  by Emilio Jesus Gallego Arias).
+- **Added:**
+  The `coq-doc` opam / Dune package will now build and install Coq's
+  documentation (`#17808 <https://github.com/coq/coq/pull/17808>`_, by
+  Emilio Jesus Gallego Arias).
+- **Added:**
+  Coq is now compatible with `memprof-limits` interruption
+  methods. This means that Coq will be recompiled when the library is
+  installed / removed from an OPAM switch.
+  (`#18906 <https://github.com/coq/coq/pull/18906>`_,
+  fixes `#17760 <https://github.com/coq/coq/issues/17760>`_,
+  by Emilio Jesus Gallego Arias).
+
+Extraction
+^^^^^^^^^^
+
+- **Added:**
+  Extension for OCaml extraction:
+  Commands to extract foreign function calls to C (external)
+  and ML function exposition (Callback.register) for calling
+  being able to call them by C functions
+  (`#18270 <https://github.com/coq/coq/pull/18270>`_,
+  fixes `#18212 <https://github.com/coq/coq/issues/18212>`_,
+  by Mario Frank).
+
+Miscellaneous
+^^^^^^^^^^^^^
+
+- **Fixed:**
+  Wrongly self-referencing extraction of primitive projections to OCaml in functors
+  (`#17321 <https://github.com/coq/coq/pull/17321>`_,
+  fixes `#16288 <https://github.com/coq/coq/issues/16288>`_,
+  by Hugo Herbelin). Note that OCaml wrappers assuming that the
+  applicative syntax of projections is provided may have
+  to use the dot notation instead.
+
 Version 8.19
 ------------
```

Comment:
This isolated Miscellaneous item could actually be moved to Extraction.

Also, some items start with a lowercase and others with an uppercase. I don't know which is best.

Reply:
There are two way to see the Changed/Added/Fixed/...
* either as the first word of the sentence (hence no second capitalization)
* as a title (hence capitalization just after)

If we don't enforce one of the two styles, we'll keep having both (not a major issue IMHO).